### ISSUE TITLE
Not working in rails 5.0.2

### ISSUE THREAD

**Issue by Radzhab** [charliesome/better_errors#363]
I use early byebyg and web_console - all work correct. At now i want use this gem, but browser return defaul rails errors pages. My settings
```
  BetterErrors.editor = :sublime
  config.consider_all_requests_local = true

  BetterErrors::Middleware.allow_ip! "0.0.0.0/0"
```

I use it from my local pc - not virtualbox, docker and similar. Why its not working?

**Comment by Sirfrummel** [comment]
Were you able to figure out a work around or fix? Just starting a new rails 5 project and I would like to use better errors with it...

**Comment by RobinDaugherty** [comment]
Sounds like the issue in #371, can you confirm?

**Comment by RobinDaugherty** [comment]
Version 2.2.0 of the gem has been released, so assuming this is the same as #371, this issue seems to be fixed.
